README.md

```diff
@@ -111,7 +111,9 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
    cd instructlab
    ```
 
-   > **NOTE:** The following steps in this document use [Python venv](https://docs.python.org/3/library/venv.html) for virtual environments. However, if you use another tool such as [pyenv](https://github.com/pyenv/pyenv) or [Conda Miniforge](https://github.com/conda-forge/miniforge) for managing Python environments on your machine continue to use that tool instead. Otherwise, you may have issues with packages that are installed but not found in `venv`.
+   > **NOTE:** The following steps in this document use [Python venv](https://docs.python.org/3/library/venv.html) for virtual environments. However, if you use another tool such as [pyenv](https://github.com/pyenv/pyenv) or [Conda Miniforge](https://github.com/conda-forge/miniforge) for managing Python environments on your machine continue to use that tool instead. Otherwise, you may have issues with packages that are installed but not found in your virtual environment.
+   <!-- -->
+   > **NOTE:** Some Python version management tools may not by default build libraries implemented in C, and CPython when installing a Python version. This can therefore result in the following error when running the `ilab data generate` command: `ModuleNotFoundError: No module named '_lzma'`. This can be resolved by building CPython during the Python installation with the `--enable-framework`. For example for `pyenv` on MacOS: `PYTHON_CONFIGURE_OPTS="--enable-framework" pyenv install 3.x`. You may need to recreate you virtual environment after reinstalling Python.
```

Comment:
> Some Python version management tools

The remainder of this note is very specific to pyenv which always builds python vs using an already built binary like conda or homebrew. I would suggest this note focus on pyenv specifically rather than trying to be more generic.

Reply:
That is a good point. It may not only be `pyenv` btw. I have updated the start sentence to specify that it is tools that build the Python binary.